### ISSUE TITLE
[Feature request] Load environment variables from .env files

### ISSUE THREAD

**Issue by tonsV2** [roboll/helmfile#1461]
The support for reqiredEnv is really handy but it would be even more so if environment variables found in .env files was included.

Docker compose has fairly advanced support for .env files. But less could do it. Simply including variables found in .env would be a good start. Please see the below link for more information.

https://docs.docker.com/compose/environment-variables/

**Comment by vsimon** [comment]
just a suggestion, for in the meantime, there is [direnv](direnv.net) + this alias gets .env files loaded at the time the helmfile command is run
`alias di='echo dotenv > .envrc && touch .env && direnv allow'`

**Comment by tonsV2** [comment]
@vsimon - Thanks for your suggestion

Minor note, isn't the touch command of the alias redundant?

direnv seems like a great project but it does seem like a minor hassel to use "plain" .env files. Not only would you have to make a symlink. Eg. ln -s .env .envrc but every environment variable needs to be prefixed with `export` which differs from the way it's done with other applications.

Furthermore, having support for this method of loading environment variables would not require direnv to be installed in a pipeline. And down the road I'd like to see this implemented supporting different environments.

**Comment by tonsV2** [comment]
https://github.com/direnv/direnv/issues/674

**Comment by vsimon** [comment]
Yes it deals with .envrc files and the need to export every variable in the .envrc file was the reason why I mentioned the bootstrapping alias (done once per folder). The `dotenv` function is a built-in direnv function that automatically sources an .env file in the same folder (an .env file without the export prefixes).

**Comment by tonsV2** [comment]
@vsimon - I see! I read dotenv as direnv. I understand the alias now. Thanks :-)
